api/src/public/login/script.js

```diff
@@ -129,16 +149,12 @@ const getSsoError = function() {
   const urlParams = new URLSearchParams(window.location.search);
   const sso_error = urlParams.get('sso_error');
 
-  if (!sso_error || ['ssoinvalid', 'ssoerror'].indexOf(sso_error) === -1) {
+  if (!sso_error || ['ssouserinvalid', 'loginerror'].indexOf(sso_error) === -1) {
```

Comment:
I could not see any reason to add the generic `ssoerror` when it seems like `loginerror` will pass through just fine and show the same error message to the user.